app/src/test/java/com/eventyay/organizer/core/presenter/AboutEventPresenterTest.java

```diff
@@ -35,7 +35,7 @@
 public class AboutEventPresenterTest {
 
     @Rule public MockitoRule mockitoRule = MockitoJUnit.rule();
-    @Mock private AboutEventVew aboutEventVew;
+    @Mock private AboutEventView aboutEventVew;
```

Comment:
Fixing the variable name `aboutEventVew` will make it perfect 😄

Reply:
Sorry, I think I missed it. Thanks for telling.

Reply:
Fixed it.